public/playerDecisionController.js

```diff
@@ -0,0 +1,121 @@
+"use strict";
+
+export default class PlayerDecisionController {
+  constructor(board) {
+    if (!Array.isArray(board.players)) {
+      throw new Error(
+        `You must provider an array with the players to the playerDecisionController constructor`
+      );
+    }
+    this.board = board;
+    this.players = this.board.players;
+    this.amountOfPlayers = this.players.length;
+
+    this.controlsContainerEle = this.board.boardContainerElem;
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+
+    this.hit = this.controlsContainerEle.querySelector(".btn--hitting");
+    this.stand = this.controlsContainerEle.querySelector(".btn--standing");
+    this.split = this.controlsContainerEle.querySelector(".btn--splitting");
+    this.double = this.controlsContainerEle.querySelector(".btn--doubling");
+
+    this.hit.addEventListener("click", this.hitting.bind(this));
+    this.stand.addEventListener("click", this.standing.bind(this));
```

Comment:
You can simplify this by calling the `nextPlayer` method directly instead of the standing method